migration/migration.go

```diff
@@ -48,6 +48,9 @@ const (
 	clusterReportTable                  = "report"
 	clusterRuleToggleTable              = "cluster_rule_toggle"
 	clusterUserRuleDisableFeedbackTable = "cluster_user_rule_disable_feedback"
+	alterTableDropColumnQuery           = "ALTER TABLE %v DROP COLUMN IF EXISTS %v"
+	alterTableAddVarcharColumn          = "ALTER TABLE %v ADD COLUMN %v VARCHAR NOT NULL DEFAULT '-1'"
```

Comment:
other migrations don't use those constants for the SQL statement, https://github.com/RedHatInsights/insights-results-aggregator/blob/master/migration/mig_0007_create_cluster_rule_toggle.go#L25  maybe we could unify this approach

Reply:
other migrations will use it in the next PRs